src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -15906,6 +15905,9 @@ def test_aks_extension_type(self, resource_group, resource_group_location):
 
         self.cmd('feature register --namespace Microsoft.KubernetesConfiguration --name ExtensionTypes')
 
+        # Wait for 5 minutes for feature registration to complete
+        time.sleep(300)
```

Comment:
I think you should be able to write a while loop that pools feature registration, rather than forcing a 5m delay here and below.

That way, once it's registered in a given sub, it's not causing a 5m delay here always.